recipes/enchant/all/configmake.h

```diff
@@ -0,0 +1,13 @@
+/* Relocation requires the INSTALLDIR define to end with a path segment that is
+ * equal to the final installation for the library file. Note that on Windows
+ * the convention is <prefix>/bin while on other platforms it's <prefix>/lib */
+#if _WIN32
+#define INSTALLDIR "/usr/local/bin"
```

Comment:
does it make any sense on Windows?

Reply:
This is only used for the relocation routines. I haven't looked too deeply at the code, but the last path segment matters here.

Reply:
yes, we need to ensure conan packages are relocatable. they couldn't contain hard-coded paths, as we don't know the path they will be installed for a particular user.
it may require an extra effort (e.g. patching), but hopefully, most of libraries are relocatable out of the box, if not by default, then at least by some environment variables.
anyway, it's worth checking what are these paths used for.